plugins/wasi_nn/wasinn_rwkv.cpp

```
@@ -0,0 +1,803 @@
+
+#define LOG_DEBUG(Debug, ...)                                                  \
+  if (Debug) {                                                                 \
+    spdlog::debug("[WASI-NN][Debug] RWKV backend: "sv __VA_ARGS__);            \
+  }
+
+#define LOG_INFO(Info, ...)                                                    \
+  if (Info) {                                                                  \
+    spdlog::info("[WASI-NN] RWKV backend: "sv __VA_ARGS__);                    \
+  }
+
+#define LOG_WARN(...) spdlog::warn("[WASI-NN] RWKV backend: "sv __VA_ARGS__);
```

Cppcheck (2.19.0)

[error] `#error` Could not find system header "<filesystem>" or "<experimental/filesystem>"

(preprocessorErrorDirective)